en/components/overlay_main.md

```diff
@@ -282,10 +276,6 @@ export class ExampleComponent {
 * [IgxOverlayService]({environment:angularApiUrl}/classes/igxoverlayservice.html)
 * [IgxOverlay Styles]({environment:sassApiUrl}/index.html#function-igx-overlay-theme)
 
-## Assumptions and Limitations
```

Comment:
why we remove this?

Reply:
I really didn't get the idea of what was written and I didn't understand what the author wanted to say :D so I removed it until I figure out what he wanted to say and how to say it better.